recipes-images/angstrom/fluke-full-image.bb

```diff
@@ -41,7 +35,7 @@ export IMAGE_BASENAME = "fluke-full-image"
 IMAGE_FSTYPES = " ubi"
 IMAGE_FSTYPES_append_fluke-cda-nighthawk = " wic.gz"
 IMAGE_FSTYPES_remove_fluke-cda-nighthawk = "ubi"
-IMAGE_FSTYPES_remove = "tar.xz"
+# IMAGE_FSTYPES_remove = "tar.xz"
```

Comment:
If this is not going to be used at any point, why not remove it all together rather than commenting it.

Reply:
Whoops that was not meant to be checked in. I needed the `tar.xz` so I can evaluate the rootfs and wasn't sure how to local extract `wic.gz`